charts/blockchain-explorer/templates/deployment.yaml

```diff
@@ -38,20 +36,20 @@ spec:
               containerPort: 25000
               protocol: TCP
           volumeMounts:
-          - name: explorer-config-volume
+          - name: config-volume
             mountPath: /app/config.json
             subPath: config.json
             readOnly: true
-          - name: explorer-config-volume
+          - name: config-volume
             mountPath: /app/.env.production
             subPath: .env.production
             readOnly: true
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
       volumes:
-      - name: explorer-config-volume
+      - name: config-volume
         configMap:
-          name: explorer-config
+          name: explorer-blockchain-explorer
```

Comment:
Hi @lukasosterheider : The name must be `{{ include "blockchain-explorer.fullname" . }}`